framework/src/base/ComputeResidualThread.C

```diff
@@ -101,7 +80,7 @@ ComputeResidualThread::onElement(const Elem *elem)
   _fe_problem.reinitMaterials(_subdomain, _tid);
 
 
-  const MooseObjectWarehouse<KernelBase> * storage;
+  MooseObjectWarehouse<KernelBase> const * storage;
```

Comment:
I have been playing around with this sort of think recently, trying to get a better handle on what order things "should", from a readability standpoint, be coded.

To me the new way makes more sense when read from right to left (which is what I usually do): `storage` is a pointer to a `const` `MooseObjectWarehouse`. The pointer is not const, the thing pointed too is.

Having the `const` at the beginning, for me, leads me to believe the pointer is `const`, which is of course not true. Because, if I create something to be `const` I put it at the beginning in general.

I am happy to change it back, it is just something I am playing with.

What does @jwpeterson say?